apiserver/DEVELOPMENT.md

```diff
@@ -20,54 +24,83 @@ The backend service can be deployed locally or within a Kubernetes cluster. The
 
 ### Pre-requisites
 
-Ensure that the admin Kubernetes configuration file is located at `~/.kube/config`.
+Ensure that the admin Kubernetes configuration file is located at `~/.kube/config`. As convenience there are two makefile targets provided to help you manage a local kind cluster:
 
-### Local Deployment
+* `make cluster` -- creates a 3 node cluster (1 control plane 2 worker) named ray-api-server-cluster
+* `make clean-cluster` -- deletes the cluster created with the `cluster` target
+
+### Local Development
 
 #### Build
 
 ```bash
-go build -a -o raymgr cmd/main.go
+#To build the executable
+make build
+
+#To start the executable build above
+../bin/kuberay-apiserver
 ```
 
-#### Start Service
+#### Test
 
 ```bash
-./raymgr
+#To run the unit tests
+make test
+```
+
+#### Start Local Service
+
+This will start the api server on your development machine. The golang race detector is turned on when starting the api server this way. It will use Kubernetes configuration file located at `~/.kube/config`. The service will not start if you do not have a connection to a Kubernetes cluster.
+
+```bash
+make run
 ```
 
 #### Access
 
-Access the service at `localhost:8888`.
+Access the service at `localhost:8888` for http, and `locahost:8887` for the RPC port.
 
 ### Kubernetes Deployment
 
-#### Build
+#### Build Image
 
 ```bash
-./docker-image-builder.sh
+#creates an image with the tag kuberay/apiserver:latest
+make docker-image 
 ```
 
-This script will build and optionally push the image to the remote Docker Hub (hub.byted.org).
-
-#### Start Service
+#### Start Kubernetes Deployment
 
 ```bash
-kubectl apply -f deploy/
-```
+#To use the  helm charts
+make deploy
 
-#### Access
+#To use the configuration from deploy/base
+make install
```

Comment:
<img width="1143" alt="Screen Shot 2023-08-08 at 6 14 34 PM" src="https://github.com/ray-project/kuberay/assets/20109646/3d5ad7c3-ccbe-4301-80e1-48867ae4e1f8">

Reply:
A bug, will fix it.

Reply:
Fixed with [c4b4272](https://github.com/ray-project/kuberay/pull/1301/commits/c4b4272b4418b4248c6163556005b03b957f7e42)